dev/release/build-release-comet.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
```

Comment:
Should we add `set -e` so that the script aborts on the first error?

Reply:
The reason I mention this is that I got many errors and then the script proceeded to attempting to build the uber jar:

```
Building binaries from https://github.com/apache/datafusion-comet.git/branch-0.2
[+] Building 1.0s (1/1) FINISHED                                                                                                                                                    
 => [internal] booting buildkit                                                                                                                                                1.0s
 => => pulling image moby/buildkit:buildx-stable-1                                                                                                                             0.7s
 => => creating container buildx_buildkit_comet-builder0                                                                                                                       0.3s
comet-builder
[+] Building 0.0s (0/0)                                                                                                                              docker-container:comet-builder
ERROR: docker exporter does not currently support exporting manifest lists
Building amd64 binary
Unable to find image 'comet-rm:latest' locally
docker: Error response from daemon: pull access denied for comet-rm, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Building arm64 binary
Unable to find image 'comet-rm:latest' locally
docker: Error response from daemon: pull access denied for comet-rm, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Building binaries completed
Copying to java build directories
Error response from daemon: No such container: comet-amd64-builder-container
Error response from daemon: No such container: comet-arm64-builder-container
Building uber jar
[INFO] Scanning for projects...
```